backend/simplemap/simplemap_test.go

```diff
@@ -0,0 +1,39 @@
+package simplemap_test
+
+import (
+	"context"
+	"github.com/heetch/confita"
+	"github.com/heetch/confita/backend/simplemap"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func TestSimplemapBackend(t *testing.T) {
+	type config struct {
+		Name     string        `config:"Name"`
+		Age      int           `config:"Age"`
+		Negative int           `config:"Negative"`
+		Balance  float64       `config:"Balance"`
+		Timeout  time.Duration `config:"Timeout"`
+	}
+
+	b := simplemap.NewBackend(map[string]interface{}{
```

Comment:
Let's have some test cases that test error cases, such as mismatched types, please.